FusionIIIT/applications/research_procedures/views.py

```diff
@@ -159,7 +168,7 @@ def add_projects(request):
                 check= HoldsDesignation.objects.filter(user__username=pid , designation__name= "Assistant Professor")
 
                 if not check.exists():
-                    messages.error(request,"Request not added, no such project investigator exists 2")
+                    messages.error(request,"Request not added, no such project investigator exists ")
                     return render(request,"rs/projects.html")  
 
         
```

Comment:
This is being repeated twice kindly take a look....

Reply:
I think one is checking for `pid` and another is checking `copid`